docs/separating-entity-definition.md

```diff
@@ -187,12 +187,60 @@ export const CategoryEntity = new EntitySchema<Category>({
 });
 ```
 
+You can use embedded entities in schema models, like this:
+```ts
+export class Name {
+    first: string;
+    last: string;
+}
+
+export const NameEntitySchema = new EntitySchema<Name>({
+    name: "name",
+    target: Name, // target field is required for Embedded Entity!
```

Comment:
It should not be required. We should be able to use entity schemas without targets for sure. Most people don't use entity schemas without classes, just with a regular types/interfaces.